src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -64,22 +70,12 @@ function ConfirmedAppointmentDetailsPage({
   confirmCancelAppointment,
   facilityData,
   fetchConfirmedAppointmentDetails,
-  confirmedStatus,
   showCancelButton,
 }) {
   const { id } = useParams();
-  const history = useHistory();
 
   useEffect(() => {
-    const status = confirmedStatus === FETCH_STATUS.succeeded;
-
-    if (!status) {
-      history.push('/');
-    }
-
-    if (!appointment || appointment.id !== id) {
-      fetchConfirmedAppointmentDetails(id);
-    }
+    fetchConfirmedAppointmentDetails(id, 'va');
```

Comment:
Always calling this because the logic for dealing with already fetched data is in the action creator